PWGHF/Tasks/taskXicc.cxx

```diff
@@ -128,6 +130,8 @@ struct HfTaskXiccMc {
     registry.add("hd0Prong0VsPtRecBg", "#Xi^{++}_{cc} (rec. unmatched) candidates;prong 0 DCAxy to prim. vertex (cm);entries", {HistType::kTH2F, {{200, -0.02, 0.02}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
     registry.add("hd0Prong1VsPtRecSig", "#Xi^{++}_{cc} (rec. matched) candidates;prong 1 DCAxy to prim. vertex (cm);entries", {HistType::kTH2F, {{200, -0.02, 0.02}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
     registry.add("hd0Prong1VsPtRecBg", "#Xi^{++}_{cc} (rec. unmatched) candidates;prong 1 DCAxy to prim. vertex (cm);entries", {HistType::kTH2F, {{200, -0.02, 0.02}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
+    registry.add("hd0d0VsPtRecSig", "#Xi^{++}_{cc} (rec. matched) candidates;product of DCAxy to prim. vertex (cm^{2}); #it{p}_{T} (GeV/#it{c})", {HistType::kTH2F, {{500, -0.05, 0.05}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
+    registry.add("hd0d0VsPtRecBg", "#Xi^{++}_{cc} (rec. unmatched) candidates;product of DCAxy to prim. vertex (cm^{2}); #it{p}_{T} (GeV/#it{c})", {HistType::kTH2F, {{500, -0.05, 0.05}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
```

Comment:
Missing `entries`.

Reply:
Again this is a TH2F and we have impact parameter products on x-axis and pT on y-axis, there is not z-axis and this is again correct.